### ISSUE TITLE
OpenBeken rebooting when simultaneously switching off two relays at the same time.

### ISSUE THREAD

**Issue by NielsPiersma** [openshwprojects/OpenBK7231T_App#923]
**Describe the bug**
When using the Show Header Toggle in HomeAssistant to power OFF two relays at the SAME time, the device will reboot. 
when toggling independently, all works fine.

**Firmware:**
Version 1.17.244
Device: LEDVANCE SMART WIFI ORBIS KITE RD 410 TW WT
Chip/model: BK7231T
Device config
P7 (PWM1) LED 1
P8 (PWM2) LED 0
FLAGS: 134218752
 
**To Reproduce**
1. Go to 'HomeAssistant'
2. Toggle the Header above the device with two relays
3. See ping disapear

**Screenshots**

![powered on](https://github.com/openshwprojects/OpenBK7231T_App/assets/70269175/be5a0392-50af-41a4-896b-572199973717)
![powered off and rebooting](https://github.com/openshwprojects/OpenBK7231T_App/assets/70269175/602973ca-5bc8-42f9-85ab-946db0f825cd)

**Additional context**
Maybe this can be solved by adding some delay for switching the secondary relay.
There is nothing logged in the log it silently reboots.
Only found it because I needed to power the lights on again after turning them of by accident.

**Comment by openshwprojects** [comment]
Hello, do you have PowerSave 1 (enabled low power mode)?

**Comment by NielsPiersma** [comment]
> Hello, do you have PowerSave 1 (enabled low power mode)?

No, I've not enabled PowerSave 1, tried explicitly with "PowerSave 0", but to no avail.

It instantly reboots when toggling both switches simultaneously.

But I think it must have something to do with the power drawn, as when I configure a bogus (chip only on a breadboard) with two relays or two leds I can switch them off both simultaneously without any problem.

The workaround is not that difficult, adding a few ms of delay in the automation and not using the header toggle keeps the device from rebooting.

Niels

**Comment by g6094199** [comment]
this sound like a power problem to me. while activating 2 relays the voltage drops to much an the device reboots.
